src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -451,3 +451,41 @@ sirius.resetMobileZoom = function () {
         document.head.removeChild(_viewportMeta);
     }
 }
+
+/**@
+ * Reads the given value from the localStorage if supported by the browser. Fails gracefully by returning null
+ * if no localStorage is available.
+ *
+ * @param name the name of the local storage variable
+ * @returns the value in the local storage variable or null
+ */
+sirius.readLocalStore = function (name) {
+    try {
+        if (isDefined(window.localStorage)) {
+            return window.localStorage.getItem('sirius.' + name);
+        }
+    } catch (e) {
+        log('Cannot read local store', {name: name, error: e});
```

Comment:
Do we wanna log this every time when the browser wont support it?

Reply:
Its the same way in oxomi.readLocalStore and oxomi.writeLocalStore.
Therefore I thought thats the way to go

Reply:
not necessarily :D